src/model.js

```diff
@@ -1461,7 +1456,7 @@ export class Model {
     let indexes = await this.queryInterface.showIndex(tableName, options);
     indexes = this._indexes.filter(item1 => !indexes.some(item2 => item1.name === item2.name)).sort((index1, index2) => {
       if (this.sequelize.options.dialect === 'postgres') {
-      // move concurrent indexes to the bottom to avoid weird deadlocks
+        // move concurrent indexes to the bottom to avoid weird deadlocks
```

Comment:
avoid introducing unrelated style changes, this will cause unnecessary conflicts with the other PRs (also valable for the other style changes)